scripts/variantstore/wdl/ngs_filter_extract.wdl

```diff
@@ -9,14 +9,19 @@ workflow NgsFilterExtract {
         File reference_index
         File reference_dict
 
-        String fq_sample_table
-        String fq_alt_allele_table
-        String query_project
+
+        String data_project
+        String? default_dataset
+
+        String query_project = data_project
```

Comment:
does this make `query_project` optional?

Reply:
In the sense that "you don't have to put it in the WDL" -- yes.

I have no idea what the difference is between

```
String? query_project = "foo"
```
and
```
String query_project = "foo"
```

In either case... if you supply the workflow input it overrides the "foo" and if you don't supply the workflow value, it doesn't complain (ie the "foo" is used).

Reply:
nice, and per Slack thread, we want it this way (and not `String?`) so that the user can't override query_project to null. 👍